### PR TITLE
TINY-7755: Update left and right arrow icons

### PR DESCRIPTION
**Pull request by PaulKutuzov**
Related Ticket:
TINY-7755

Description of Changes:
* Updated arrow-left and arrow-right icons

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)

Review:
* [ ] Milestone set
* [ ] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

**Review by ashfordneil — APPROVED** [review]
Thanks for the PR Paul! Two small things:
- [x] We generally prefer branch names like `feature/TINY-7755` instead of `origin/TINY-7755` (don't worry about changing it for this PR though, it's not a big deal)
- [ ] Can you please edit the PR title and change it to something like "TINY-7755: Update left and right arrow icons"

File: modules/tinymce/CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Icons with a `-rtl` suffix in their name will now automatically be used when the UI is rendered in right-to-left mode #TINY-7782
 
 ### Changed
+- Updated arrow-right and arrow-left icons #TINY-7755
```
Comment (JamesToohey) [inline modules/tinymce/CHANGELOG.md:41]:
```suggestion
- Updated the `arrow-right` and `arrow-left` icons #TINY-7755
```

**Review by JamesToohey — APPROVED** [review]
Looks good Paul. To add to what Neil said - the PR title is what the commit will be titled when the branch is squashed before merging, that's why it's important.

You'll also need to sign the CLA. Check your spam folder if you haven't received the email

**Review by lostkeys — APPROVED** [review]

**Review by HAFRMO — APPROVED** [review]

**Comment by lnewson** [comment]
I'm going to convert this back to a draft so we stop getting pinged about this, as there was discussions/concerns raised from Engineering about updating these icons instead of the `action-next`/`action-previous` icons which is what Design had told us were to be used for this previously.

**Comment by TheSpyder** [comment]
Looks like this was shelved internally

**Pull request closed without merging**